src/DssSpell.sol

```diff
@@ -38,24 +38,41 @@ contract DssSpellAction is DssAction {
     //
 
     uint256 constant THOUSAND = 10**3;
-    uint256 constant MILLION  = 10**6;
+    uint256 constant MILLION = 10**6;
+    uint256 constant BILLION = 10**9;
+
+    // Core Units Budget Addresses
+    address constant INTERIM_MULTISIG = 0x73f09254a81e1F835Ee442d1b3262c1f1d7A13ff;
+    address constant RISK = 0xd98ef20520048a35EdA9A202137847A62120d2d9;
 
     // Disable Office Hours
     function officeHours() public override returns (bool) {
         return false;
     }
 
     function actions() public override {
-        // Set bump parameter from 10,000 to 30,000
-        DssExecLib.setSurplusAuctionAmount(30 * THOUSAND);
+        // Increase the COMP-A Maximum Debt Ceiling from 10M to 30M
+        DssExecLib.setIlkAutoLineDebtCeiling("COMP-A", 30 * MILLION);
+
+        // Increase the ZRX-A Maximum Debt Ceiling from 5M to 10M
+        DssExecLib.setIlkAutoLineDebtCeiling("ZRX-A", 10 * MILLION);
+
+        // Increase the YFI-A Maximum Debt Ceiling from 45M to 75M
+        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 75 * MILLION);
+
+        // Increase the PSM-USDC-A Debt Ceiling from 1B to 2B
+        DssExecLib.increaseIlkDebtCeiling("PSM-USDC-A", 1 * BILLION, true);
+
+        // Provide Core Unit Budgets
+
+        // Real-World Finance (Interim Multi-Sig)
+        DssExecLib.sendPaymentFromSurplusBuffer(INTERIM_MULTISIG, 40 * THOUSAND);
 
-        // Set ETH-B dust parameter from 2,000 to 15,000
-        DssExecLib.setIlkMinVaultAmount("ETH-B", 15 * THOUSAND);
+        // Governance (Interim Multi-Sig)
+        DssExecLib.sendPaymentFromSurplusBuffer(INTERIM_MULTISIG, 80 * THOUSAND);
```

Comment:
Was there a specific request to do it in two separate transfers?
Maybe is better for explicitness, but we could also do `(40 + 80) * THOUSAND` in one transfer.